presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,434 @@ Except for the limit on queued queries, when a resource group runs out of a reso
 it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
-The resource groups and associated selection rules are configured by a manager which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+In PrestoDB, Resource Groups are a powerful tool for managing query execution and resource allocation.
+They allow administrators to control how resources are allocated and utilized on a Presto cluster.
```

Comment:
```suggestion
They allow administrators to control how resources are allocated and utilized on a Presto cluster.

```

Reply:
A line break needs to be present before the text of a header. When I build this page locally Resource Group Limits, and its formatting line of dashes underneath, looks like this. 

<img width="950" alt="Screenshot 2023-09-20 at 1 33 18 PM" src="https://github.com/prestodb/presto/assets/7013443/99d8c69c-7d51-497f-93b3-fdfbc1cfcd76">

Please build locally and review the formatting.

Reply:
Fixed, thanks!